.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   tests:
     name: Release
     runs-on: ubuntu-latest
-
+    if: github.repository == 'appwrite-appwrite'
```

Comment:
Shouldnt this be appwrite/appwrite?

Reply:
Ah yes you're correct! Allow me to fix that!